datafusion/sqllogictest/test_files/window.slt

```diff
@@ -833,19 +833,13 @@ LIMIT 5
 
 
 #fn window_frame_ranges_preceding_and_preceding
-query II
+query error DataFusion error: Arrow error: Invalid argument error: Column 'count\(\*\) ORDER BY \[aggregate_test_100\.c2 ASC NULLS LAST\] RANGE BETWEEN 3 PRECEDING AND 1 PRECEDING' is declared as non\-nullable but contains null values
```

Comment:
Is it the same as the ignored test above? If so, should this be ignored too?

Reply:
There is no 'ignore' in SLT, so, as I understand, this one is effectively ignored by expecting the error?

Reply:
it's the same commit as ignored tests. there is a test that -- when starting to fail (ie during update) -- will ask for the whole commit to be reverted, and then this gets reverted as well